sklearn/preprocessing/_data.py

```diff
@@ -3318,6 +3318,8 @@ def _more_tags(self):
 
 def power_transform(X, method="yeo-johnson", *, standardize=True, copy=True):
     """
+    Family of parametric, monotonic transformations to make data more Gaussian-like.
+    
```

Comment:
This line has (invisible) extra white-spaces. This causes our black linter to trip:

https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=39376&view=logs&j=32e2e1bb-a28f-5b18-6cfc-3f01273f5609&t=fc67071d-c3d4-58b8-d38e-cafc0d3c731a

```suggestion

```

To avoid this kind of problems, it is recommended 

```
pip install black==22.1.0
```
and then:

```
black sklearn/preprocessing/_data.py
```

Alternatively, you can also call black to format your code directly in VS Code using `ctrl-shift-p` + "Format document" or "alt-shift-f" directly.